client/common/cmdline.c

```diff
@@ -474,6 +475,22 @@ BOOL freerdp_client_print_command_line_help_ex(int argc, char** argv,
 	printf("\n");
 	printf("Drive Redirection: /drive:home,/home/user\n");
 	printf("Smartcard Redirection: /smartcard:<device>\n");
+	printf("Smartcard logon with rdp only:                /smartcard-logon [/sec:rdp]\n");
```

Comment:
This is misleading, main method is `/sec:tls` (as `/sec:rdp` is deprecated)

Reply:
Hum yeah will remove that.